src/widgets/_toolbars.scss

```diff
@@ -15,6 +15,17 @@ actionbar {
     &.flat{
         @extend .inline-toolbar;
     }
+
+    &.title {
+        font-size: 11pt;
```

Comment:
does using `pt` units scale when the user changes their font size?

Also do you know where this is used? Is there a reason these are large than the window title? That seems a little odd

Reply:
`pt` is meant to be scalable in the context of fonts only.

Also, some apps add labels with a title/subtitle styling to actionbars, and these are larger to accomodate the styling seen in their screenshots.

Reply:
Can you elaborate on what you mean by "some apps"? A specific example would help here.

I'm not really worried about breaking screenshots since this is an entirely new stylesheet and it will break screenshots regardless

Reply:
Yeah, I think that it's maybe best to not write for edge cases then. An example was Synaptic; but since it's meant to be a cleaner stylesheet, I'll remove this.